packages/altive_lints/example/test/avoid_hardcoded_japanese_test.dart

```diff
@@ -0,0 +1,9 @@
+// Check the `avoid_hardcoded_japanese` rule.
+//
+// It should exclude warnings for the entire `/test/` directory.
```

Comment:
![badge](https://img.shields.io/badge/review-nits-inactive)
It is not necessary to enclose the directory with `/` since it is specified as a “directory”.　

```suggestion
// It should exclude warnings for the entire `test` directory.
```